portfolio/src/main/webapp/script.js

```diff
@@ -57,9 +72,18 @@ function deleteComments() {
   fetch(request).then(getComments());
 }
 
+function checkPassword() {
+  var adminInput = document.getElementById("admin-password").value;
+  if (adminInput == "azby88") {
+    document.getElementById("removeAllComments").style.visibility = "visible";
+  } else {
+    document.getElementById("removeAllComments").style.visibility = "hidden";
+  }
+}
```

Comment:
That's really a great feature! would you please hide the admin password either in a file you don't add here (same thing you already did for the API key) or have it in the datastore and let the server check for that in a new servlet by comparing the user input with the one in storage?

Reply:
Thank you, that's the plan - it was next on my list to add it with the API files.